apps/emqx_management/src/emqx_mgmt_api_stats.erl

```diff
@@ -129,7 +129,19 @@ list(get, #{query_string := Qs}) ->
         _ ->
             Data = [
                 maps:from_list(emqx_mgmt:get_stats(Node) ++ [{node, Node}])
-             || Node <- mria:running_nodes()
+             || Node <- running_nodes()
             ],
             {200, Data}
     end.
+
+%%%==============================================================================================
+%% Internal
+
+running_nodes() ->
+    Nodes = erlang:nodes([visible, this]),
+    RpcResults = erpc:multicall(Nodes, emqx, is_running, [], 15000),
```

Comment:
I think this should be added to a `emqx_mgmt_api_stats_proto_v1` module for backplane communications?

Reply:
@thalesmg like this?

Reply:
Yes.  I think the bpapi static check will also probably ask to add this new module to `apps/emqx/priv/bpapi.versions`.